manifests/server/pg_hba_rule.pp

```diff
@@ -12,10 +12,12 @@
 
   # Needed for testing primarily, support for multiple files is not really
   # working.
-  $target      = $postgresql::server::pg_hba_conf_path
+  $target             = $postgresql::server::pg_hba_conf_path,
+  $postgresql_version = $postgresql::server::_version,
+  $manage_pg_hba_conf = $postgresql::server::manage_pg_hba_conf
```

Comment:
Doesn't the parse-time order dependency already exist?  I'm just trying to give myself a way to use this defined type without being tightly coupled to `postgresql::server`